packages/terra-table/src/subcomponents/Section.jsx

```diff
@@ -162,37 +168,118 @@ function Section(props) {
     onSectionSelect(id);
   }, [id, onSectionSelect]);
 
+  const sectionHeader = (
+    <tbody>
+      {/* Render section rows */}
+      {!isHidden && (
+      <tr
+        aria-rowindex={sectionRowIndex}
+        className={cx('header-row', theme.className)}
+        data-section-id={id}
+      >
+        <th
+          id={`${tableId}-${id}`}
+          className={cx('header-cell')}
+          align="left"
+          colSpan={displayedColumns.length}
+          role="columnheader"
+          scope="col"
+          tabIndex={isGridContext && !hasSectionButton ? -1 : undefined}
+        >
+          <SectionHeader
+            className={cx('section-header')}
+            text={text}
+            isOpen={hasSectionButton ? !isCollapsed : undefined}
+            onClick={hasSectionButton ? handleClick : undefined}
+            isTitleSticky
+            boundedWidth={boundingWidth}
+          />
+        </th>
+      </tr>
+      )}
+    </tbody>
+  );
+
+  const rowProps = {
+    displayedColumns,
+    firstRowId,
+    height: rowHeight,
+    isTableStriped,
+    lastRowId,
+    onCellSelect,
+    rowHeaderIndex,
+    rowMinimumHeight,
+    rowSelectionMode,
+    sectionId: !isHidden ? id : undefined,
+    tableId,
+  };
+
+  if (subsections) {
+    return (
+      <>
+        {sectionHeader}
+        {!isCollapsed && subsections.map((subsection) => (
+          <>
+            <tbody className={cx('subsection', {
+              collapsed: isCollapsed,
+              collapsible: isCollapsible,
+            }, theme.className)}
+            >
+
+              <tr
+                aria-rowindex={subsection.subSectionRowIndex}
+                className={cx('header-row', theme.className)}
+                data-section-id={id}
+              >
+                <th
+                  id={`${tableId}-${id}-${subsection.id}`}
+                  className={cx('header-cell')}
+                  align="left"
+                  colSpan={displayedColumns.length}
+                  role="columnheader"
+                  scope="col"
+                  tabIndex={isGridContext ? -1 : undefined}
+                >
+
+                  <SectionHeader
+                    className={cx('subsection')}
```

Comment:
Shouldn't this class be something like subsection-header-row to match the section classes?

Reply:
This appends the `subsection` to the existing `header-row` class, so it would be `header-row subsection`, but I can rename it if it's preferred.